entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt

```diff
@@ -65,6 +58,10 @@ class EntityHandler(
         val newEntityUri = entityService.createEntity(ngsiLdEntity)
         authorizationService.createAdminLink(newEntityUri, userId)
 
+        entitiesEventService.publishEntityEvent(
```

Comment:
Out this PR scope : I think we should go towards a more DDD approach for Domain Events publishing http://www.kamilgrzybek.com/design/how-to-publish-and-handle-domain-events/ and not our HTTP Adapter